### ISSUE TITLE
Card gets stuck whenever you drag it and your thumb slides off the screen.

### ISSUE THREAD

**Issue by itsdevcoffee** [meteor-factory/react-native-tinder-swipe-cards#57]
So I have essentially the exact same code as the example. It works perfect unless your thumb goes off the screen at the left or right edge of the phone while swiping. I'm not sure if anyone else has experienced something similar and if so how did you fix it.

[Video example](https://youtu.be/1fsu4KaKfO4)

**Comment by juergengunz** [comment]
yes unfortunately I have the same exact issue and it ruins this cool swiper :(

**Comment by juergengunz** [comment]
On iOS the App freezes and on Android the app crashed with following error:
<img width="473" alt="bildschirmfoto 2017-02-23 um 21 59 01" src="https://cloud.githubusercontent.com/assets/12918614/23278965/bbcf2ebe-fa13-11e6-9d4f-9e7ec945fafe.png">

**Comment by juergengunz** [comment]
this fixes it https://github.com/meteor-factory/react-native-tinder-swipe-cards/pull/59/files

**Comment by lolobosse** [comment]
It needs to be released to NPM right?

**Comment by itsdevcoffee** [comment]
Thanks so much man. Worked like a charm.

**Comment by derekbrown** [comment]
@vitorebatista Is the fix for this being released to NPM soon?
